app/views/admin/articles/article_blocks/_show_medium.html.slim

```diff
@@ -1,7 +1,6 @@
 .box-body
-  - if medium.attachment.attached?
-    - if medium.image?
-      = image_tag medium.image_url(:thumb), class: 'img-thumbnail'
+  - if medium.image.present?
+    = image_tag medium.image.file.md.url, class: 'img-thumbnail'
```

Comment:
__[Rails Best Practices]__

law of demeter


<!-- Comment by [SideCI](https://sideci.com) -->